### PR TITLE
Index Mississippi river area of JRC's monthly water cover dataset 

### LINKED ISSUE #81: JRC monthly water VRT + STAC
We're exporting the monthly recognized water datasets used in JRC calculations from GEE and need to make this data consumable for Raster Vision

~~- [ ] generate VRT join disparate tifs~~ (no longer necessary due to downsizing of export)
- [x] index ~~VRTs~~ tifs via a STAC

### PR DESCRIPTION
**Pull request by moradology**
# Overview

Provides a STAC which catalogs all monthly JRC global water cover data in and around the Mississippi river. Sourced from [GEE](https://developers.google.com/earth-engine/datasets/catalog/JRC_GSW1_2_MonthlyHistory) as the data is otherwise not publicly available

Here's the generated catalog:

[global-water-monthly-catalog.tar.gz](https://github.com/azavea/noaa-flood-mapping/files/5263083/global-water-monthly-catalog.tar.gz)

Closes #81

### CODE REVIEW

**Review by CloudNiner — CHANGES_REQUESTED** [review]
Needs docstring fixed.

Also, IIUC, this is specifically a catalog of Mississippi River JRC events so perhaps we shouldn't put it in a `global-water-monthly` folder? I'd guess `jrc-monthly-water-mississippi` or something similar to that would be more descriptive of what the folder actually contains?

**Comment by moradology** [comment]
Good catch RE the docstring. As for the S3 bucket, I did create a subfolder. Do you think a top level bucket is the better choice?

**Comment by CloudNiner** [comment]
> As for the S3 bucket

Oh. Sorry, wasn't clear. I was referring to the local directory the scripts are stored in. `./catalogs/global-water-monthly`.

No preference for where the files go on S3, as long as its documented. For our other catalogs, I believe that's mostly documented in README.md files in the same dir as the catalog's main.sh.

File: catalogs/mississippi-surface-water/build_catalog.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+if __name__ == "__main__":
+    """Constructs STAC Catalog for surface water observed on a monthly basis around the Mississippi river
+
+    Will include all images with the extension `tif`
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--jrc-monthly-root-s3",
+        required=True,
+        help="Bucket+key path to a directory containing GLOFIMR flood ids",
```
Comment (CloudNiner) [inline catalogs/mississippi-surface-water/build_catalog.py]: I believe this docstring is also still incorrect.

**Review by CloudNiner — APPROVED** [review]
Works great, update that one last help string before merge.

File: catalogs/mississippi-surface-water/build_catalog.py (hunk 1 above)
Comment (moradology) [reply catalogs/mississippi-surface-water/build_catalog.py]: Oof. Fixed now

**Pull request merged**